editor/import/resource_importer_scene.cpp

```diff
@@ -2013,6 +2036,14 @@ Node *ResourceImporterScene::_generate_meshes(Node *p_node, const Dictionary &p_
 		mesh_node->set_transform(src_mesh_node->get_transform());
 		mesh_node->set_skin(src_mesh_node->get_skin());
 		mesh_node->set_skeleton_path(src_mesh_node->get_skeleton_path());
+
+		//copy meta from imported mesh
```

Comment:
```suggestion
		// Copy meta from imported mesh.
```

Reply:
Done, but the file predominantly uses no space after //